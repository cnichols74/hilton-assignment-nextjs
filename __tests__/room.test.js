import React from 'react'
import { shallow } from 'enzyme'
import Room from '../components/room'
import data from '../data/data'

test('should render a Room Component', () => {
  const wrapper = shallow(<Room room={data.rooms[0]}/>)
  expect(wrapper).toMatchSnapshot()
})
