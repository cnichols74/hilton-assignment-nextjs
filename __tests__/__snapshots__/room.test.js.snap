// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Room Component 1`] = `
<div
  className="jsx-2164095659 room-form-container"
>
  <p
    className="jsx-2164095659 room-form-title"
  >
    <label
      className="jsx-2164095659"
    >
      <input
        className="jsx-2164095659"
        onChange={[Function]}
        type="checkbox"
      />
       Room
    </label>
     
  </p>
  <form
    className="jsx-2164095659 room-form"
    style={null}
  >
    <fieldset
      className="jsx-2164095659"
      disabled={true}
    >
      <span
        className="jsx-2164095659"
      >
        Adults
      </span>
      <span
        className="jsx-2164095659"
      >
        Children
      </span>
      <div
        className="jsx-2164095659"
      >
        <span
          className="jsx-2164095659"
        >
          (18+)
        </span>
        <span
          className="jsx-2164095659"
        >
          (0-17)
        </span>
      </div>
      <div
        className="jsx-2164095659"
      >
        <Select
          name="Adults"
          onChange={[Function]}
        />
        <Select
          name="Children"
          onChange={[Function]}
        />
      </div>
    </fieldset>
  </form>
  <JSXStyle
    id="2164095659"
  >
    input[type='checkbox'].jsx-2164095659{-ms-transform:scale(1.5);-moz-transform:scale(1.5);-webkit-transform:scale(1.5);-o-transform:scale(1.5);position:relative;top:-2px;padding:10px;}.room-form-title.jsx-2164095659{position:relative;font-size:18px;font-weight:bold;left:23px;}fieldset.jsx-2164095659{border:none;}span.jsx-2164095659{display:inline-block;width:96px;}.room-form-container.jsx-2164095659{background:#eee;margin:20px 18px;padding:0px 13px;border:1px solid #eee;border-radius:10px;box-shadow:2px 3px 10px 2px #ccc;}.room-form.jsx-2164095659{margin:1px 1px 7px;padding:35px 30px;border-radius:0 0px 5px 5px;}
  </JSXStyle>
</div>
`;
